src/CLI/CLI.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Main class to register WP-CLI commands.
+ *
+ * @package WooCommerce\Migrator\CLI
+ */
+
+namespace WooCommerce\Migrator\CLI;
+
+use WP_CLI;
+
+/**
+ * Main class for registering the WP-CLI commands.
+ */
+class CLI {
+
+	/**
+	 * The command registrar.
+	 *
+	 * @var Command_Registrar
+	 */
+	private $registrar;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Command_Registrar $registrar The command registrar.
+	 */
+	public function __construct( Command_Registrar $registrar ) {
+		$this->registrar = $registrar;
+	}
+
+	/**
+	 * Registers the WP-CLI commands.
+	 */
+	public function register_commands() {
+		if ( ! class_exists( 'WP_CLI' ) ) {
```

Comment:
Nit. This seems a bit excessive, considering we already test this in the main file.

Reply:
fair point, addressed here: https://github.com/woocommerce/WooCommerce-Migrator/pull/3/commits/d2ef4a9c9cfeaf9dc006ea4085d8d15e28421e22